ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/OntologiesViewImpl.java

```diff
@@ -455,21 +455,41 @@ public void updateButtonStatus() {
         deleteButton.setEnabled(selectedOntology != null && selectedOntology != activeOntology);
         deleteHierarchy.setEnabled(selectedOntology != null && editorTree.getSelectionModel().getSelectedItem() != null);
         categorize.setEnabled(selectedOntology != null && targetApp != null && !targetApp.getAppType().equalsIgnoreCase(App.EXTERNAL_APP));
-        deleteApp.setEnabled(selectedOntology != null && targetApp != null && !targetApp.getAppType().equalsIgnoreCase(App.EXTERNAL_APP));
+        deleteApp.setEnabled(selectedOntology != null && targetApp != null && !targetApp.getAppType().equalsIgnoreCase(App.EXTERNAL_APP) && !targetApp.isDeleted());
```

Comment:
Not necessary for this PR, but I would consider refactoring these conditionals into booleans.
It might make this section a little more readable.
For example
```java
boolean ontologySelected = selectedOntology != null;
boolean appSelected = targetApp != null;
boolean appExternal = targetApp.getAppType().equalsIgnoreCase(App.EXTERNAL_APP);
boolean appDeleted = targetApp.isDeleted();
...
deleteApp.setEnabled(ontologySelected && appSelected && !appExternal && !appDeleted);
```
These could even be named with an `is` prefix (isAppSelected) if preferred.